test/unit/rules/no-nested-splattributes-test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/no-nested-splattributes').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-nested-splattributes',
+
+  config: true,
+
+  good: [
+    '<div ...attributes>...</div>',
+    '<div ...attributes>...</div><div ...attributes>...</div>',
+  ],
```

Comment:
Can you add a good entry that doesn't have any `...attributes`. Pretty much doesn't matter (given how the rule works), but doesn't hurt to have either 😸 

```suggestion
  good: [
    '<div ...attributes>...</div>',
    '<div ...attributes>...</div><div ...attributes>...</div>',
    '<div></div>',
  ],
```

Reply:
done ✅